### PR TITLE
Docs for #282: added oci registry readme help text

### PR DESCRIPTION
**Pull request by jessesanford**

### CODE REVIEW

**Comment by omrishiv** [comment]
Is it worth mentioning how to pull the image from the cluster? I assume we either need a service account with the docker credentials or create a .dockerconfig on the node?

File: README.md [hunk 1]
```diff
@@ -218,6 +219,112 @@ You can also view the updated Application spec by going to this address: https:/
 The second package directory defines two normal ArgoCD applications referencing a remote repository.
 They are applied as-is.
 
+## Local OCI Registry
+
+The local Gitea instance created by idpbuilder contains a built in OCI registry for hosting container images as "packages" in Gitea nomenclature.
+
+It is a standard OCI registry, so the API should be compatible with any tools that are OCI compliant. That includes the `docker` cli.
+
```
Comment (csantanapr) [inline README.md:227]: Make a warning here that to login username is case sensitive `giteaAdmin` but when using username/org in OCI image url needs be lower case `giteaadmin` (noticed there are 2 `a`)

I use `giteaAdmin` in docker tag, and lost a lot of time why is not working

File: README.md [hunk 2]
```diff
@@ -218,6 +219,112 @@ You can also view the updated Application spec by going to this address: https:/
...
+For example you can push an image by running:
+
+```bash
+docker login gitea.cnoe.localtest.me:8443
```
Comment (csantanapr) [inline README.md:231]: In addition give a one line command example how to login like
```
idpbuilder get secrets -p gitea -o json | jq '.[0].data.password' -r | docker login -u giteaAdmin --password-stdin gitea.cnoe.localtest.me:8443
```

**Review by csantanapr — CHANGES_REQUESTED** [review]

**Comment by csantanapr** [comment]
@omrishiv 
>Awesome, thanks! Is it worth mentioning how to pull the image from the cluster? I assume we either need a service account with the docker credentials or create a .dockerconfig on the node?

no need for pull secrets, the images are public read access

**Comment by omrishiv** [comment]
Ah, thanks for the clarification. Maybe something to think about in the future, but good to know.

File: README.md (hunk 1 above)
Comment (nimakaviani) [reply README.md:227]: this is a good call. not sure why we need to call it `giteaAdmin` where `admin` would be good enough. I will create an issue for it so we can change it.

**Pull request merged**